### PR TITLE
Update govt-urls for M-22-09

### PR DESCRIPTION
**Pull request by h-m-f-t**
[M-22-09](https://zerotrust.cyber.gov/federal-zero-trust-strategy/#4-safely-making-applications-internet-accessible) requires GSA and CISA to collaborate on cataloging non-.gov domain names. This PR updates the govt-urls page with new language.

TODO
* [x] Add Touchpoints form
* [x] Update email address
* [x] Add template

### CODE REVIEW

**Comment by dawnpm** [comment]
@h-m-f-t please change the target for this pr to be https://github.com/GSA/search-gov-website/tree/dawnpm-update-non-gov-urls-instructions, that will allow us to do a run through preview before merging to main.

File: about/policy/govt-urls.md [hunk 1]
```diff
@@ -1,34 +1,88 @@
...
 
-In support of USA.gov and [M-17-06 &ndash; Policies for Federal Agency Public Websites and Digital Services](https://www.whitehouse.gov/wp-content/uploads/legacy_drupal_files/omb/memoranda/2017/m-17-06.pdf), Search.gov maintains a list of all public government domains that don't end in .gov or .mil.
+The list is used by several U.S. government services, including USA.gov to support their all-of-government search, and the .gov registry to support an inventory of government domains.
+
+Anyone can submit a domain to the list.
```
Comment (dawnpm) [inline about/policy/govt-urls.md:19]: Please update to
Anyone can [submit](#non-gov-domain-collector) a domain to the list.

File: about/policy/govt-urls.md [hunk 2]
```diff
@@ -1,34 +1,88 @@
...
+
+## What domains are in scope of this list?
+
+All non-.gov and non-.mil domain names registered by your agency, or by a contractor on behalf of your agency, must be included in this list. This includes domains that aren\'t even used on the internet but are merely registered.
+
+| **Domain Description** | **Example** |
+|------------------------|-------------|
+| Registered to facilitate redirects | * **agency.com** which may redirect to agency.gov |
+| Serve some infrastructure purpose, even if not public-facing | * **agencyapi.com** <br> * **agency-dashboard.net** |
```
Comment (dawnpm) [inline about/policy/govt-urls.md:28]: `<br />`

File: about/policy/govt-urls.md [hunk 3]
```diff
@@ -1,34 +1,88 @@
...
+| Registered by a contractor on behalf of the agency | * **agencycampaignsite.us** |
+| Registered long ago and is not expired | * **legacyagency.org** |
+| Registered to prevent others from obtaining it (sometimes called "defensive registrations", which includes "typo" domains) | * **ouragency.com** <br> * **0uragency.com** <br> * **houragency.com** |
```
Comment (dawnpm) [inline about/policy/govt-urls.md:31]: `<br />`

File: about/policy/govt-urls.md [hunk 4]
```diff
@@ -1,34 +1,88 @@
...
 
-In support of USA.gov and [M-17-06 &ndash; Policies for Federal Agency Public Websites and Digital Services](https://www.whitehouse.gov/wp-content/uploads/legacy_drupal_files/omb/memoranda/2017/m-17-06.pdf), Search.gov maintains a list of all public government domains that don't end in .gov or .mil.
+The list is used by several U.S. government services, including USA.gov to support their all-of-government search, and the .gov registry to support an inventory of government domains.
+
+Anyone can submit a domain to the list.
+
+## What domains are in scope of this list?
+
+All non-.gov and non-.mil domain names registered by your agency, or by a contractor on behalf of your agency, must be included in this list. This includes domains that aren\'t even used on the internet but are merely registered.
```
Comment (dawnpm) [inline about/policy/govt-urls.md:23]: Curious why you escaped the `'`?

Comment (h-m-f-t) [reply about/policy/govt-urls.md:23]: Pandoc artifact

File: about/policy/govt-urls.md [hunk 5]
```diff
@@ -1,34 +1,88 @@
...
+
+| **Domain Description** | **Example** |
+|------------------------|-------------|
+| Registered to facilitate redirects | * **agency.com** which may redirect to agency.gov |
+| Serve some infrastructure purpose, even if not public-facing | * **agencyapi.com** <br> * **agency-dashboard.net** |
+| Registered by a contractor on behalf of the agency | * **agencycampaignsite.us** |
+| Registered long ago and is not expired | * **legacyagency.org** |
+| Registered to prevent others from obtaining it (sometimes called "defensive registrations", which includes "typo" domains) | * **ouragency.com** <br> * **0uragency.com** <br> * **houragency.com** |
+
+Do not add individual subdomains or hostnames within that domain. For example, report *agency.com*, but do not also report *saas.agency.com*.
```
Comment (dawnpm) [inline about/policy/govt-urls.md:33]: This got messed up in the final changes to the doc. The `that` doesn't have a reference anymore. How about 
`Do not add individual subdomains or hostnames within a domain that's already on the list.`

File: about/policy/govt-urls.md [hunk 6]
```diff
@@ -1,34 +1,88 @@
...
+
+Report a "subdomain" if (1) your agency has a domain registered on a country-code top-level domain (or other "[public suffixes](https://publicsuffix.org/)") or (2) your agency publishes federal information on nongovernmental domains that you do not operate.
```
Comment (dawnpm) [inline about/policy/govt-urls.md:35]: let's make this "public suffix" - singular

File: about/policy/govt-urls.md [hunk 7]
```diff
@@ -1,34 +1,88 @@
...
+
+Generally, report a domain if it is registered pursuant to the terms of a grant and used for federal purposes, but do not report domains registered by grantees which your agency does not oversee or technologically support.
+
+## What domains are out of scope of this list?
+
+This list **excludes** the domains of third-party online services where the agency is merely a user or a customer, not the domain's registrant.
+
+| **Domain Description** | **Example** |
+| -----------------------|-------------|
+| Software-as-a-service sites | * **agency-my.sharepoint.com** <br> * **agency-service.slack.com** |
```
Comment (dawnpm) [inline about/policy/govt-urls.md]: `<br />`

File: about/policy/govt-urls.md [hunk 8]
```diff
@@ -1,34 +1,88 @@
...
+| Cloud-managed resources | * **s3.amazonaws.com** |
+| Social media sites | * **twitter.com** <br> * **linkedin.com** |
```
Comment (dawnpm) [inline about/policy/govt-urls.md]: `<br />`

File: about/policy/govt-urls.md [hunk 9]
```diff
@@ -1,34 +1,88 @@
...
+| Code repositories | * **github.com/agency** |
 
 ## How to Update the List
 
-Federal agencies [are required](https://www.whitehouse.gov/sites/whitehouse.gov/files/omb/memoranda/2017/m-17-06.pdf) to submit to Search.gov all non-.gov websites for inclusion in the list. This includes subdomains of a second-level domain managed by a third party, and federally controlled subfolders of a domain managed by a third party.
+1.  **Gather domain names**. Domains are sometimes confused for "websites", so consider asking for a "list of websites/domain registrations we manage" from the following types of roles:
+
```
Comment (dawnpm) [inline about/policy/govt-urls.md:60]: remove line

File: about/policy/govt-urls.md [hunk 10]
```diff
@@ -1,34 +1,88 @@
...
+    a.  External affairs, web publishing, or press team
+
```
Comment (dawnpm) [inline about/policy/govt-urls.md:62]: remove line

File: about/policy/govt-urls.md [hunk 11]
```diff
@@ -1,34 +1,88 @@
...
+    b.  DNS managers, network/security operations
+
```
Comment (dawnpm) [inline about/policy/govt-urls.md:64]: remove line

File: about/policy/govt-urls.md [hunk 12]
```diff
@@ -1,34 +1,88 @@
...
+    c.  Budget or accounting teams who might process domain registration payments to registrars like GoDaddy, Network Solutions, Namecheap, etc.
+
+> If you have questions about whether a certain domain should be included, [open an issue](https://github.com/GSA/govt-urls/issues) in Github or [email us](mailto:search@support.digitalgov.gov).
+
+2.  **Send your updates.** We will review all submissions before including them on the list.
+
+-   For *multiple additions* or any *changes:*
+
```
Comment (dawnpm) [inline about/policy/govt-urls.md:72]: remove line

File: about/policy/govt-urls.md [hunk 13]
```diff
@@ -1,34 +1,88 @@
...
+| Cloud-managed resources | * **s3.amazonaws.com** |
+| Social media sites | * **twitter.com** <br> * **linkedin.com** |
+| Code repositories | * **github.com/agency** |
 
 ## How to Update the List
 
-Federal agencies [are required](https://www.whitehouse.gov/sites/whitehouse.gov/files/omb/memoranda/2017/m-17-06.pdf) to submit to Search.gov all non-.gov websites for inclusion in the list. This includes subdomains of a second-level domain managed by a third party, and federally controlled subfolders of a domain managed by a third party.
+1.  **Gather domain names**. Domains are sometimes confused for "websites", so consider asking for a "list of websites/domain registrations we manage" from the following types of roles:
+
+    a.  External affairs, web publishing, or press team
```
Comment (dawnpm) [inline about/policy/govt-urls.md:61]: GSA style guide requires periods at the end of list items.

Comment (h-m-f-t) [reply about/policy/govt-urls.md:61]: Would you like a period after "etc."?

**Review by dawnpm — APPROVED** [review]

**Pull request merged**